eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -49,6 +52,8 @@ if [ "$STAGE" = "end2end" ]; then
    run_e2e_test '' 'cd node_tests && npm run test_operator && npm run test_ui'
 elif [ "$STAGE" = "debug" ]; then
    run_e2e_test '-ti' 'bash'
+elif [ "$STAGE" = "smoke" ]; then
+   run_smoke_test '' 'cd node_tests && npm run test_smoke'
```

Comment:
could this be appended to the `end2end` tests instead of being added as a new stage step?

Reply:
I can but its not really end-2-end
WOuld you recommend it to be added int he same step even though its not end-2-end?